innerprod/fullysec/damgard_multi_test.go

```diff
@@ -26,17 +26,23 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestFullySec_DamgardMultiDDH(t *testing.T) {
+func testFullySec_DamgardMultiDDHFromParam(t *testing.T, param damgardTestParam) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [don't use underscores in Go names; func testFullySec_DamgardMultiDDHFromParam should be testFullySecDamgardMultiDDHFromParam](https://app.codacy.com/app/tilenmarc/gofe_2/pullRequest?prid=4432579)